### ISSUE TITLE
Does harbor support using Ceph or Glusterfs as the backend for image storage

### ISSUE THREAD

**Issue by gyliu513** [vmware/harbor#2263]
Hi, this is just a question about harbor, does harbor support using Ceph or Glusterfs as the backend for image storage?

**Comment by reasonerjt** [comment]
This really depends on the support of upstream project  docker/distribution

**Comment by gyliu513** [comment]
Yes, so if I can have distributed storage with harbor to store the images, then I will not need the replica feature? How does the replica feature works with distributed storage as backend as the distributed storage backend can replica the images automatically.

**Comment by reasonerjt** [comment]
The replication feature is for replicate images across Harbor instance.  
They may use different storage services, for example Harbor A may use filesystem, Harbor B may use S3, and the specific storage service is transparent to Harbor when image on Harbor A is replicated to Harbor B.

**Comment by gyliu513** [comment]
Yes, but my case is if I have two harbor clusters as HA cluster, and want the two clusters share one Ceph cluster as the image store, is this doable?

**Comment by hainingzhang** [comment]
what did you mean by two clusters of Harbor? If you meant two instances, yes, Harbor can support two instances share the same Ceph cluster. @gyliu513

**Comment by fenghui2013** [comment]
@gyliu513 What is the ceph and harbor version? Use swift to access ceph?
